contrib/ansible/templates/containerd.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=containerd container runtime
+Documentation=https://containerd.io
+After=network.target
+
+[Service]
```

Comment:
At least make it the same with `contrib/systemd-units`. We need systemd to restart containerd for us.
It seems that restart sec could be shorter.
We also need `OOMScoreAdjust=-999` for containerd, please update `contrib/systemd-units`, too.

Reply:
using relative path for now. Lets see if it becomes too much of a hassle, will think about alternative.

Reply:
We should use the systemd units file inside the tarball. I'll do a it in a following up PR.